config/spring.rb

```diff
@@ -1,6 +1,6 @@
-%w(
+%w[
   .ruby-version
   .rbenv-vars
   tmp/restart.txt
   tmp/caching-dev.txt
-).each { |path| Spring.watch(path) }
+].each { |path| Spring.watch(path) }
```

Comment:
Same question: what was the violation Rubocop raised here?
Just curious...

Reply:
Guessing the same thing, rubocop -a?